build.gradle

```diff
@@ -1,279 +1,40 @@
-/*
```

Comment:
suggest re-instating this file - rather than have to go through this again. The template version is much closer in structure than this version (which was based on our old repos)